include/revng/Pipes/TaggedFunctionKind.h

```diff
@@ -54,6 +55,16 @@ class TaggedFunctionKind : public pipeline::LLVMKind {
 
   void appendAllTargets(const pipeline::Context &Ctx,
                         pipeline::TargetsList &Out) const override;
+
+  static cppcoro::generator<
+    std::pair<const model::Function *, llvm::Function *>>
+  getLLVMAndCommit(pipeline::ExecutionContext &Context,
+                   llvm::Module &Module,
+                   llvm::StringRef ContainerName);
+
+  static cppcoro::generator<const model::Function *>
+  getAndCommit(pipeline::ExecutionContext &Context,
+               const pipeline::ContainerBase &Container);
 };
 
```

Comment:
I'd rename both of these into ``getFunctionsAndCommit`.
The arguments automatically disambiguate what's happening.